versions.props

```diff
@@ -35,11 +35,10 @@ io.airlift:airline = 0.8
 io.atomix.catalyst:* = 1.1.2
 io.atomix.copycat:* = 1.2.0
 io.atomix:* = 1.0.0-rc9
-io.dropwizard.metrics:metrics-* = 3.2.3
-io.dropwizard.modules:* = 0.9.0-1
-io.dropwizard:* = 0.9.3
+io.dropwizard.metrics:metrics-* = 4.0.5
```

Comment:
just spoke to infra team, this is not safe to do and it has been explicitly pinned to the last 3.x version. Apparently there's a 5.x version (to be released) which has different package names so they can coexist etc.